### PR TITLE
Fix libcal booking

### PR DESCRIPTION
**Pull request by vcai122**

### CODE REVIEW

**Review by judtinzhang — APPROVED** [review]
LOL good stuff thanks vincent!

File: backend/gsr_booking/models.py [hunk 1]
```diff
@@ -96,8 +101,13 @@ class GSR(models.Model):
     name = models.CharField(max_length=255)
     image_url = models.URLField()
 
+    in_use = models.BooleanField(default=True)
+
+    objects = GSRManager()
+    all_objects = models.Manager()  # for admin page
```
Comment (judtinzhang) [inline backend/gsr_booking/models.py:107]: that is great, thanks vincent! @tuneerroy @ashleyzhang01 @JeffersonDing @dr-Jess just for reference look at this man!

File: backend/tests/gsr_booking/test_gsr_views.py [hunk 2]
```diff
@@ -82,8 +82,8 @@ def test_recent(self):
         response = self.client.get(reverse("recent-gsrs"))
         res_json = json.loads(response.content)
         self.assertEquals(2, len(res_json))
-        self.assertEquals(6, len(res_json[0]))
-        self.assertEquals(6, len(res_json[1]))
+        self.assertEquals(7, len(res_json[0]))
+        self.assertEquals(7, len(res_json[1]))
```
Comment (judtinzhang) [inline backend/tests/gsr_booking/test_gsr_views.py:86]: i'm imagining that the extra field is the `in_use`? any reason it still shows up in the route despite all of our queries using `.object` to query? if this is the case, I think we should remove this field from showing up

**Review by judtinzhang — COMMENTED** [review]
One quick question before push

File: backend/tests/gsr_booking/test_gsr_views.py (hunk 2 above)
Comment (vcai122) [reply backend/tests/gsr_booking/test_gsr_views.py]: oh the serializer was still serializing all the fields. fixed now

**Review by judtinzhang — APPROVED** [review]

File: backend/tests/gsr_booking/test_gsr_views.py (hunk 2 above)
Comment (judtinzhang) [reply backend/tests/gsr_booking/test_gsr_views.py]: bet LGTM

**Pull request merged**